onboarding_auth/onboardingAuthData/src/main/java/com/alexafit/onboardingauthdata/di/OnboardingDataModule.kt

```diff
@@ -0,0 +1,53 @@
+package com.alexafit.onboardingauthdata.di
+
+import com.alexafit.onboardingauthdata.remote.TaskyApi
+import com.alexafit.onboardingauthdata.repository.OnboardingAuthRepository
+import com.alexafit.onboardingauthdata.repository.OnboardingAuthRepositoryImpl
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.create
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object OnboardingDataModule {
```

Comment:
Is there any onboarding on the app tho?

Reply:
I umbrella login with onboarding because if you need to register an account than that could technically be considered onboarding. Naming could have been Authorization to not be so general